views/items/index.pug

```diff
@@ -4,6 +4,11 @@ block content
   Items
 
   form(method="post", action="/items")
+    p Add a ToDo
```

Comment:
Please use bootstrap form styles instead of paragraph tags http://getbootstrap.com/css/#forms